pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -639,7 +639,16 @@ in
     buildInputs = [ args.snappy ];
   };
 
-  sqlite3 = attrs: {
+  sqlite3 = attrs: if lib.versionAtLeast attrs.version "1.5.0"
+  then {
+    nativeBuildInputs = [ pkg-config sqlite.dev ];
+    buildInputs = [ sqlite.out ];
+    buildFlags = [
+      "--enable-system-libraries"
+      "--with-pkg-config=${pkg-config}/bin/pkg-config"
```

Comment:
I suspect this usage of pkg-config is not cross fields.

Reply:
I also don't think so. Is a full path needed here? Otherwise $PKG_CONFIG might work too, maybe?

Reply:
Could you explain what "is not cross fields." means here?

Anyway, I did confirm the entry is not needed. I got confused because `--with-pkg-config` on its own was failing :( But we seem to be ok without it.

Reply:
> Could you explain what "is not cross fields." means here?

*friendly